src/vmm/src/guest_config/aarch64_templates/g3g2.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use arch::aarch64::regs::{ID_AA64ISAR0_EL1, ID_AA64ISAR1_EL1, ID_AA64MMFR2_EL1, ID_AA64PFR0_EL1};
+
+use super::{CpuTemplateAarch64, RegisterValue};
+
+/// Aarch64 template to mask graviton3 as graviton2
+pub fn g3g2() -> CpuTemplateAarch64 {
+    CpuTemplateAarch64(
+        vec![
+            (
+                // Disable dit CPU feature.
+                ID_AA64PFR0_EL1,
+                RegisterValue {
+                    mask: 0xFFF0FFFFFFFFFFFF,
+                    value: 0,
+                },
+            ),
+            (
+                ID_AA64ISAR0_EL1,
+                // Disable sha3 CPU feature. sha3 occupies bits [35:32] in ID_AA64ISAR0_EL1
+                // register. "If the value of SHA2 field is 0b0010,
+                // ID_AA64ISAR0_EL1.SHA3 must have the value 0b0001
+                // So when disabling sha3, we also need to set sha2 to 0b0001.
+                // disable sha3
+                //
+                // set sha2 to disable sha3
+                //
+                // Disable sm3 and sm4 CPU features.
+                // "This field (sm3) must have the same value as ID_AA64ISAR0_EL1.SM4."
+                //
+                // Disable asimdfhm (fhm) CPU feature.
+                //
+                // Disable flagm (ts) CPU feature.
```

Comment:
These comments were initially accompanied by the specified operations. Let's rephrase these to make a little more sense.

Reply:
done